mwclient/client.py

```diff
@@ -969,7 +969,21 @@ def ask(self, query, title=None):
         API doc: https://semantic-mediawiki.org/wiki/Ask_API
 
         Returns:
-            Generator for retrieving all search results
+            Generator for retrieving all search results.
+            Iterating over the generator gives a dictionary containing the answers
+            to the query.
+            The dictionary is empty if either:
+               - the query is valid but there are no answers,
+               - or if the query is not valid.
```

Comment:
Please update the comment to reflect that the dictionary is empty of if there is no answers